tests/unit/conftest.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for Jenkins-k8s-operator charm unit tests."""
+
+from io import StringIO
+from pathlib import Path
+from secrets import token_hex
+from typing import Any, BinaryIO, Optional, TextIO, Union
+from unittest.mock import MagicMock
+
+import pytest
+import requests
+from ops.model import Container
+from ops.testing import Harness
+
+from charm import JenkinsK8SOperatorCharm
+from jenkins import JENKINS_PASSWORD_FILE_PATH
+from types_ import Credentials
+
+from .helpers import make_relative_to_path
+from .types_ import ContainerWithPath
+
+
+@pytest.fixture(scope="function", name="harness")
+def harness_fixture():
+    """Enable ops test framework harness."""
+    harness = Harness(JenkinsK8SOperatorCharm)
+    yield harness
+    harness.cleanup()
+
+
+@pytest.fixture(scope="function", name="jenkins_version")
+def jenkins_version_fixture():
+    """Jenkins version fixture."""
+    return "2.400"
```

Comment:
I think this is defined twice (also in the rockcraft?) Is there any way we could have the Jenkins version in just 1 place?

Reply:
Updated to refer from rockcraft yaml, thanks!